core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -1058,6 +1073,28 @@ private void handleGetUserListFailure(String errorCode, String errorMessage, Str
         }
     }
 
+    /**
+     * This method is responsible for calling the relevant methods when there is a failure while trying to get the
+     * user list.
+     *
+     * @param errorCode    Error Code.
+     * @param errorMessage Error Message.
+     * @param claim        Claim URI.
+     * @param claimValue   Claim Value.
+     * @throws UserStoreException Exception that will be thrown by relevant listner methods.
+     */
+    private void handleGetUserCountFailure(String errorCode, String errorMessage, String claim, String claimValue
+    ) throws UserStoreException {
+
+        for (UserManagementErrorEventListener listener : UMListenerServiceComponent
+                .getUserManagementErrorEventListeners()) {
+            if (listener.isEnable() && !listener
+                    .onGetUserListFailure(errorCode, errorMessage, claim, claimValue, this)) {
```

Comment:
why are you calling the onGetUserListFailure method when there is a failure in counting the users?

Reply:
fixed

Reply:
fixed